Epsilon.Host.Frontend/src/components/TopNavigation.vue

```diff
@@ -37,6 +37,7 @@ const selectedTerm: Ref<EnrollmentTerm | undefined> = ref(undefined)
 
 watch(selectedTerm, () => {
     store.commit("setCurrentTerm", selectedTerm.value)
+    store.commit("correctCurrentTerm")
```

Comment:
correctCurrentTerm seems like a bit of a side effect, don't we just want to always only store the correct term. and thus just the term. If not, maybe re name it to something that indicates what it does, like what range it has or something along those lines. What I'm saying is that I don't know what a correct term is, so it may cause confusion to others.

Reply:
correctCurrentTerm, The therm needs to be correct for every current term so that the right data is shown. The term definitions of Canvas aka Fontys don't align with submitted data from students. So data is not shown in the right semester. thats why. 

About storing the right data is something that I find a bit too much and redundant. Now all the data that is stored is created by the backend. 

The bast what would then be to change the data in the backend.

Reply:
Make separate issue for new team members